cmd/chunks-inspect/loki.go

```diff
@@ -43,7 +43,7 @@ var (
 	enclz4_1M   = Encoding{code: 6, name: "lz4-1M", readerFn: func(reader io.Reader) (io.Reader, error) { return lz4.NewReader(reader), nil }}
 	enclz4_4M   = Encoding{code: 7, name: "lz4-4M", readerFn: func(reader io.Reader) (io.Reader, error) { return lz4.NewReader(reader), nil }}
 	encFlate    = Encoding{code: 8, name: "flate", readerFn: func(reader io.Reader) (io.Reader, error) { return flate.NewReader(reader), nil }}
-	encZstd     = Encoding{code: 9, name: "lz4-256k", readerFn: func(reader io.Reader) (io.Reader, error) {
+	encZstd     = Encoding{code: 9, name: "zstd", readerFn: func(reader io.Reader) (io.Reader, error) {
```

Comment:
doesn't that mean that we won't be inspecting `lz4-256k` chunks anyway? I'm not sure if `lzy4-256` are compatible but in case they are, maybe the best thing here would be to duplicate this to have both supports.

Reply:
I think this was just a copy & paste error. A decoder for `lz4-256k` is added in [line 42](https://github.com/grafana/loki/blob/43c11a9a287afba695f7502367d1b7c41ab4e612/cmd/chunks-inspect/loki.go#L42).

Reply:
ohhh good call, thank you. Approved!